docs/config/lua/keyassignment/PromptInputLine.md

```diff
@@ -18,6 +18,8 @@ upon the input.
   objects from the current pane and window, and `line` is the text that the
   user entered. `line` may be `nil` if they hit Escape without entering
   anything, or CTRL-C to cancel the input.
+* `prompt` - the text to show as the prompt. You may embed escape sequences
+  and/or use [wezterm.format](../wezterm/format.md).  Defaults to: `"> "`
```

Comment:
let's annotate this with the upcoming version:

```suggestion
  and/or use [wezterm.format](../wezterm/format.md).  Defaults to: `"> "`. {{since('nightly', inline=True)}}
```